Dockerfile

```diff
@@ -1,9 +1,12 @@
 FROM python:3-slim
 WORKDIR /app
 
+ARG VERSION="0.0.0"
+ENV SETUPTOOLS_SCM_PRETEND_VERSION_FOR_TELEGRAM_SPOTTED_DMI_BOT=${VERSION}
```

Comment:
this env name seems a bit long hehe

Reply:
Unfortunately, it's not up to us: it's taken from the [documentation](https://setuptools-scm.readthedocs.io/en/latest/usage/#with-dockerpodman).